CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+# Contributing to Science on schema.org
+
+The goal of science-on-schema.org is to foster community agreement on guidelines for using schema.org for science-related resource descriptions. Thus, we welcome and encourage community contributions to the work, including proposals for new guidelines, clarifications on the existing guidelines, and proposals for extensions to the schema.org model to accomodate new fields.  Thus, community members are invited to contribute through:
+
+- Creating and discussing proposals through [GitHub issues](https://github.com/ESIPFed/science-on-schema.org/issues)
+- Implementing proposed features by editing the guidelines and extension documentation, and editing the Architectural Decision Record for the proposal
+- Issuing a Pull Request to request review of the proposed changes, and to discuss the feature and iterate on development
+
+# Development and Release process
+Because people will review our guidance documents online, it will be helpful to be explicit about versioning and releases of the guidance and other documents.  In particular, in GitHub, users will normally see the `master` branch, which therefore should reflect the current stable release of the documentation (rather thanb confusing people with in-progress proposed changes that are not yet released).
+
+Consequently, we will use a [GitFlow](https://nvie.com/posts/a-successful-git-branching-model/)-inspired release model in which the `master` branch always reflects the current stable release, a `develop` branch is used for merging finished proposals being prepared for release, and `feature` branches are used for creating changes to implement specific proposals that are reflected in an Architectural Decision Record (ADR).  For changes that do not require a formal decision via and ADR, such as spelling corrections, grammatical rewording, etc., maintainers can commit changes directly to the `develop` branch, and other contributors can do a pull request directly against the `develop` branch.  The use of feature branches is really focused on managing proposals that need discussion, review, and a decision through an ADR. Maintainers will make judgement calls on whether an ADR is needed, and might convert contributed pull requests to a feature branch if they determine that an ADR is needed.
```

Comment:
... and `feature` branches ~~are used for creating changes to~~ **for** implement**ing** specific ...

... decision via an~~d~~ ADR, ...